lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -387,10 +420,15 @@ def main():
     state = module.params.get('state')
     purge_rules = module.params['purge_rules']
     purge_rules_egress = module.params['purge_rules_egress']
+    tags = module.params['tags']
 
     if state == 'present' and not description:
         module.fail_json(msg='Must provide description when state is present.')
 
+    nonstring_tags = [k for k, v in tags.items() if type(v) != str]
```

Comment:
add `from ansible.module_utils.six import string_types` to the imports and then do here `isinstance(v, string_types)`